.gitlab-ci.yml

```
@@ -654,10 +654,32 @@ library:ci-argosy:
 library:ci-bbv:
   extends: .ci-template
 
+library:ci-coqutil:
+  extends: .ci-template-flambda
+
+library:ci-kami:
+  extends: .ci-template-flambda
+
+library:ci-coq_record_update:
+  extends: .ci-template-flambda
+
+library:ci-riscv_coq:
+  extends: .ci-template
```

This one as well should be:
```suggestion
  extends: .ci-template-flambda
```